client-src/elements/chromedash-feature-detail.js

```diff
@@ -466,7 +467,7 @@ class ChromedashFeatureDetail extends LitElement {
         @sl-after-show=${this.updateCollapsed}
         @sl-after-hide=${this.updateCollapsed}
         ?open=${isActive || defaultOpen}
-        class=${isActive ? 'active' : ''}
+        class="${isActive ? 'active' : ''} ${isStage ? 'stage' : ''}"
```

Comment:
How would it look like if it is both active and stage? class="activestage"?

Reply:
It has whitespace between the expressions, so it becomes `class="active stage"` which works.  That also means that that space is always there, but it does hurt anything.